torch/_dynamo/variables/user_defined.py

```diff
@@ -1393,6 +1418,36 @@ def call_method(
         )
 
 
+class UserDefinedExceptionObjectVariable(UserDefinedObjectVariable):
```

Comment:
Is this a variable tracker for user-defined exceptions instances? Is it OK that this is not inheriting from ExceptionVariable? @anijain2305